test/trento/application/integration/discovery/protocol/enrich_register_cluster_host_test.exs

```diff
@@ -9,7 +9,7 @@ defmodule Trento.EnrichRegisterClusterHostTest do
 
   alias Trento.Support.Middleware.Enrichable
 
-  alias Trento.ClusterEnrichmentData
+  alias Trento.Clusters.ClusterEnrichmentData
```

Comment:
You need to change the test module as well:
`Trento.Clusters.EnrichRegisterClusterHostTest` I guess.
Anyway, let's better wait until the integration PR is merged

Reply:
I see PR #2022 used `Trento.Infrastructure.Commanded.Middleware.EnrichRegisterClusterHostTest`, I guess I don't need to address this